lua/nvim-treesitter/ts_utils.lua

```diff
@@ -134,7 +134,7 @@ function M.get_node_at_cursor(winnr)
   if not root_lang_tree then
     return
   end
-  local root = M.get_root_for_position(cursor_range[1], cursor_range[2], root_lang_tree)
+  local root = M.get_root_for_position(cursor_range[1], cursor_range[2], root_lang_tree, ignore_injected_langs)
```

Comment:
just return `root_lang_tree` here if `ignore_injected_langs`

Reply:
it was slightly more complex that just assigning `root_lang_tree` since we need iterate over its trees to get the `root` but i've done this